legos/cryptocurrency.py

```diff
@@ -2,15 +2,28 @@
 import requests
 import logging
 import json
+import configparser
 
 logger = logging.getLogger(__name__)
 
 
 class Cryptocurrency(Lego):
+    def __init__(self, baseplate, lock):
+        super().__init__(baseplate, lock)
+        self.crypto_index = configparser.ConfigParser()
+        self.crypto_index.read('crypto_index.ini')
```

Comment:
If you want to use a config file, you should check for presence first. I recommend making a configure method to handle this part.

Reply:
Addressed by 59347c9ebbfd67a3a14f36b7543e6dc9b33c5fa3